starwars/src/App.js

```diff
@@ -29,10 +30,17 @@ class App extends Component {
       });
   };
 
+  
+  
+
   render() {
+    console.log(this.state.starwarsChars);
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        {this.state.starwarsChars.map(char => {
+          return <Character name={char.name} height={char.height} mass={char.mass} hair_color={char.hair_color} skin_color={char.skin_color} films={char.films} key={Math.random()}   />
+        })}
```

Comment:
On this line, rather than passing all the object properties in one at a time, you can just pass the entire char object and access the properties from the child component. 

`
<Character char={char}/>
`
This makes the program more robust, because you can add properties to the Character and they will be passed down to the child component.